### DISCUSSION TITLE
can we just skip enhancing some classes or jar files

### DISCUSSION THREAD

**Discussion by lingli0219** [apache/skywalking#8653] [Q&A] [answered]
When we use HierarchyMatch.byHierarchyMatch(someInterface), can we skip enhancing some classes or jar files? I did not find the solution to this. Can someone help me with this?

**Comment by wu-sheng** [comment, marked as answer]
There is no such thing about `jar files` in binary code processing on default. Skipping some classes is possible, but you need to write your own match implementation, which you could learn from ByteBuddy and the existing matcher.

**Reply by lingli0219** [reply]
OK. Thank you.
